docs/developing/publishing-apps.md

```diff
@@ -131,6 +131,8 @@ Your app's manifest, or package definition file, (`sandstorm-pkgdef.capnp`) cont
 
 You can embed both SVGs or PNGs, and Sandstorm will use the best version provided for the use in question. Using PNGs requires a slightly different structure, which you can find an example of [here](https://github.com/dwrensha/sharelatex/blob/sandstorm-app/sandstorm-pkgdef.capnp#L26).
 
+<img src="https://inozs8hbcttx5fdveukc.oasis.sandstorm.io/docs/docs-package-icons.png"></img>
```

Comment:
If possible, I have some suggestions here:

- Use one image tag per image, so that it breaks apart (which IMHO it should) on narrow screens

- Use Markdown image syntax, like in docs/administering/faq.md